creator-node/src/middlewares.js

```diff
@@ -150,36 +153,55 @@ async function _getCreatorNodeEndpoints (req, wallet) {
   if (config.get('isUserMetadataNode')) throw new Error('Not available for userMetadataNode')
   const libs = req.app.get('audiusLibs')
 
+  req.logger.info(`Starting _getCreatorNodeEndpoints for wallet ${wallet}`)
+  const start = Date.now()
+
   // Poll discprov until it has indexed provided blocknumber to ensure up-to-date user data.
-  let user
-  const maxRetries = 10
-  let retries = 0
+  let user = null
   const { blockNumber } = req.body
   if (blockNumber) {
```

Comment:
super nit but if you do !blocknumber it'd be much easier to read without the else clause at the bottom